compiler/rustc_errors/src/diagnostic_builder.rs

```diff
@@ -198,6 +198,45 @@ impl EmissionGuarantee for () {
     }
 }
 
+impl<'a> DiagnosticBuilder<'a, !> {
+    /// Convenience function for internal use, clients should use one of the
+    /// `struct_*` methods on [`Handler`].
+    crate fn new_fatal(handler: &'a Handler, level: Level, message: &str) -> Self {
```

Comment:
`new_fatal` with level begin `Level::Fatal` feels redundant to me. What's the reason for providing the `level` argument instead of hardcoding it to `Level::Fatal`?

Reply:
copypasta is the reason, I can fix